### PR TITLE
bandwhich: Update to 0.22.2

### PR DESCRIPTION
**Pull request by nazar-stasiv**
**Summary**
- Generate completion & manpage
- Log unresolved processes in more detail + general refactor
- Display bandwidth in different unit families
- Show interface names
- Table formatting logic overhaul
- Make logging race-free using a global lock & macro
- Use once_cell::sync::Lazy to make regex usage more ergonomic
- Fix vague CLI option documentation

**Test Plan**
- watch IP connections on network interface

**Checklist**
- [X] Package was built and tested against unstable

### CODE REVIEW

File: packages/b/bandwhich/package.yml [hunk 1]
```diff
@@ -17,4 +17,3 @@ build      : |
     cargo build --release    
 install    : |
     install -Dm00755 target/release/bandwhich $installdir/usr/bin/bandwhich
-    install -Dm00644 docs/bandwhich.1 -t $installdir/usr/share/man/man1/
```
Comment (davidjharder) [inline packages/b/bandwhich/package.yml:20]: Why was this removed? The changelogs make it sound like the man.1 file should be available

**Review by davidjharder — CHANGES_REQUESTED** [review]

**Comment by davidjharder** [comment]
I'm a little baffled. I don't see generated manpages when I basically copy Arch's build steps: https://gitlab.archlinux.org/archlinux/packaging/packages/bandwhich/-/blob/main/PKGBUILD?ref_type=heads

**Comment by nazar-stasiv** [comment]
Fixed man page

**Review by davidjharder — APPROVED** [review]
Great, thanks
